tests/amqp_management_ut/amqp_management_ut.c

```diff
@@ -270,6 +272,38 @@ static ASYNC_OPERATION_HANDLE my_messagesender_send_async(MESSAGE_SENDER_HANDLE
     return test_send_operation;
 }
 
+static void ASYNC_OPERATION_HANDLE_ToString(char* string, size_t bufferSize, ASYNC_OPERATION_HANDLE val)
+{
+    (void)bufferSize;
+    (void)sprintf(string, "%p", val);
+}
+
+static int ASYNC_OPERATION_HANDLE_Compare(ASYNC_OPERATION_HANDLE left, ASYNC_OPERATION_HANDLE right)
+{
+    return left != right;
+}
+
```

Comment:
I do not really see any new unit test here. It can't be that we don't want to test the fact that the cancel handler removes the item from the list.
And this is the easiest level to test it.

Reply:
Unit tests added.